vumi/message.py

```diff
@@ -219,6 +231,8 @@ def process_fields(self, fields):
         fields.setdefault('content', None)
         fields.setdefault('transport_metadata', {})
         fields.setdefault('group', None)
+        fields.setdefault('to_addr_type', self.AT_MSISDN)
+        fields.setdefault('from_addr_type', self.AT_MSISDN)
```

Comment:
Agreed. We can let the thing handling the message decide what a sensible default is.